redaxo/src/core/lang/de_de.lang

```diff
@@ -52,6 +52,9 @@ login_welcome = Bitte anmelden.
 login_logged_out = Abgemeldet.
 stay_logged_in = Eingeloggt bleiben
 
+login_impersonate = Wechseln
```

Comment:
Ich finde das deutsche label nicht passend. Hab aber keinen besseren Vorschlag

Reply:
Ich suche da schon seit Jahren nach was besserem, nutze das bisher aber immer in meinen Symfony-Apps, und habe mich dran gewöhnt ;)
Bin aber für andere Vorschläge offen.